rectangle.cpp

```diff
@@ -12,53 +12,30 @@
 
 #include <memory>
 
-Rectangle::Rectangle(const glm::vec3& position, float width, float height) :
-    mPosition {position},
+static const std::array<unsigned int, 6> kIndicies[] = {
```

Comment:
I think the empty braces `[]` are not needed here. You may actually be making a c array of std::arrays, and if that's the case god help us all

Reply:
Yeah good catch! I think that was left over from when they were c style arrays